.github/workflows/build.yml

```diff
@@ -3,10 +3,10 @@ name: Continuous Integration
 on:
   push:
     branches:
-      - '*'
```

Comment:
Make the change in `main` branch to update this apart (temp fix in this PR to make it work)